main.tf

```diff
@@ -1,8 +1,17 @@
+data "external" "md5sum" {
+  program = ["sh", "${path.module}/md5sum.sh"]
```

Comment:
Per the TF documentation: https://www.terraform.io/docs/providers/external/data_source.html

> This mechanism is provided as an "escape hatch" for exceptional situations where a first-class Terraform provider is not more appropriate.

Terraform natively supports compression. We use it other modules. Use this instead if a shell script.

Reply:
https://www.terraform.io/docs/providers/archive/d/archive_file.html